data/translations/ko/state-data.json

```diff
@@ -3,310 +3,410 @@
     "confirm_registration_link": "https://www.mvp.sos.ga.gov/MVP/mvp.do"
   },
   "gu": {
     "hp_link": "https://gec.guam.gov/",
     "registration_link": "https://gec.guam.gov/register/",
     "more_info_link": "https://gec.guam.gov/",
-    "online_deadline": "선거일 10일 전까지",
-    "mail_received_deadline": "반드시 선거일 10일 전까지 까지 접수해야 합니다.",
-    "ip_deadline": "선거일 10일 전까지",
+    "ip_deadline": "2022년 10월 28일 금요일",
+    "mail_received_deadline": "반드시 2022년 10월 28일 금요일 까지 접수되어야 합니다.",
+    "default_online_deadline": "선거일 10일 전까지",
+    "default_mail_received_deadline": "반드시 선거일 10일 전까지 까지 접수해야 합니다.",
+    "default_ip_deadline": "선거일 10일 전까지",
     "confirm_registration_link": "https://gec.guam.gov/validate/"
   },
   "hi": {
     "hp_link": "https://elections.hawaii.gov/",
     "registration_link": "https://olvr.hawaii.gov/",
     "more_info_link": "https://elections.hawaii.gov/voters/registration/",
-    "ip_deadline": "선거당일까지 등록이 가능합니다.",
-    "online_deadline": "선거일 30일 전까지",
-    "mail_received_deadline": "반드시 선거일 30일 전까지 까지 접수해야 합니다.",
+    "ip_deadline": "2022년 11월 8일 화요일",
+    "online_deadline": "2022년 11월 8일 화요일",
+    "mail_postmarked_deadline": "반드시 2022년 10월 31일 월요일 까지 소인이 찍혀야 합니다.",
+    "default_ip_deadline": "선거당일까지 등록이 가능합니다.",
+    "default_online_deadline": "선거일 30일 전까지",
+    "default_mail_postmarked_deadline": "반드시 선거일 30일 전까지 까지 접수해야 합니다.",
     "confirm_registration_link": "https://olvr.hawaii.gov/"
   },
   "id": {
     "hp_link": "https://idahovotes.gov/",
     "registration_link": "https://elections.sos.idaho.gov/ElectionLink/ElectionLink/ApplicationInstructions.aspx",
     "more_info_link": "https://idahovotes.gov/voting/",
-    "ip_deadline": "선거당일까지 등록이 가능합니다.",
-    "online_deadline": "선거일 25일 전까지",
-    "mail_postmarked_deadline": "반드시 선거일 25일 전까지 일자로 소인이 날인되어야 합니다.",
+    "ip_deadline": "2022년 11월 8일 화요일",
+    "online_deadline": "2022년 10일 14일 금요일",
+    "mail_postmarked_deadline": "반드시 2022년 10일 14일 금요일 까지 소인이 찍혀야 합니다.",
+    "default_ip_deadline": "선거당일까지 등록이 가능합니다.",
+    "default_online_deadline": "선거일 25일 전까지",
+    "default_mail_postmarked_deadline": "반드시 선거일 25일 전까지 일자로 소인이 날인되어야 합니다.",
     "confirm_registration_link": "https://elections.sos.idaho.gov/ElectionLink/ElectionLink/VoterSearch.aspx"
   },
   "il": {
     "hp_link": "https://www.elections.il.gov/",
     "registration_link": "https://ova.elections.il.gov/",
     "more_info_link": "https://www.elections.il.gov/Default.aspx",
-    "ip_deadline": "선거당일까지 등록이 가능합니다.",
-    "online_deadline": "선거일 16일 전까지",
-    "mail_received_deadline": "반드시 선거일 28일 전까지 까지 접수해야 합니다.",
+    "ip_deadline": "2022년 11월 8일 화요일",
+    "online_deadline": "2022년 10월 18일 화요일",
+    "mail_postmarked_deadline": "반드시 2022년 10월 6일 목요일 까지 소인이 찍혀야 합니다.",
+    "default_ip_deadline": "선거당일까지 등록이 가능합니다.",
+    "default_online_deadline": "선거일 16일 전까지",
+    "default_mail_received_deadline": "반드시 선거일 28일 전까지 까지 접수해야 합니다.",
```

Comment:
Update to `default_mail_postmarked_deadline`

Reply:
fixed